addon/lib/document.js

```diff
@@ -185,7 +185,14 @@ export default EmberObject.extend({
       }
 
       const visibleFields = this.fields.filter(f => !f.question.hidden);
-      if (visibleFields.every(f => f.isValid)) {
+      const requiredFields = visibleFields.filter(
+        f => f.question.isRequired !== "false"
```

Comment:
Isn't `question.isRequired` a JEXL expression?

Reply:
From it being a string; that's possible. Let me check if I can find the right property.

Reply:
As discussed, should be `optional`

Reply:
Done